src/ompl/base/ProblemDefinition.h

```diff
@@ -442,10 +462,16 @@ namespace ompl
             /// @endcond
 
             // flag to either save or not save approximate solutions
-            bool acceptApproximate_ {true};
+            bool acceptApproximate_ {false};
```

Comment:
shouldn't change the default in class definition

Reply:
This flag was added by us

Reply:
still why don't you change the value where it needs to be changed?

Reply:
The default should be false since we never want approximate solutions